prometheus/go_collector.go

```diff
@@ -236,6 +242,9 @@ func (c *goCollector) Describe(ch chan<- *Desc) {
 func (c *goCollector) Collect(ch chan<- Metric) {
 	c.goroutines.Set(float64(runtime.NumGoroutine()))
 	ch <- c.goroutines
+	n, _ := runtime.ThreadCreateProfile(nil)
+	c.threads.Set(float64(n))
```

Comment:
![image](https://cloud.githubusercontent.com/assets/4769989/22914925/87ab3c5c-f2ae-
11e6-8896-865e20ed9bf6.png)

I want to add thoese counts too, I just post a new metric for any suggestion.